src/renderer/PresetsBar.tsx

```diff
@@ -48,7 +28,7 @@ const PresetsBar = () => {
   const [selectedPresetName, setSelectedPresetName] = useState<
     string | undefined
   >(undefined);
-
+  const [presetNameError, setPresetNameError] = useState<string>('');
```

Comment:
Personally, i got a little confused between this error and the existing preset rename error. Maybe call this `newPresetNameError`?